### PR TITLE
Update development dependencies to latest

### PR DESCRIPTION
**Pull request by drKraken**
Seem like now are using not latest package versions. Suggest to update development dependencies to latest versions of packages.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+    "gulp-jscs": "^1.4.0",
+    "gulp-jshint": "^1.9.2",
+    "gulp-load-plugins": "^0.8.1",
     "gulp-rename": "^1.2.0",
-    "gulp-replace": "^0.5.0",
-    "jquery": "1.11.2",
-    "jshint-stylish": "^1.0.0",
-    "mocha": "^2.0.1",
+    "gulp-replace": "^0.5.3",
+    "jquery": "2.1.3",
```
Comment (arthurvr) [inline package.json]: We don't want to update to jQuery 2.x, the 2.x versions don't support IE8.

**Comment by arthurvr** [comment]
@drKraken Thanks for the many contributions to h5bp projects lately!

**Comment by alrra** [comment]
Thanks @drKraken!

**Comment by drKraken** [comment]
@arthurvr thank you guys for great projects! :+1:

**Pull request closed without merging**